source/Application/Controller/ArticleDetailsController.php

```diff
@@ -962,19 +962,19 @@ public function getAlsoBoughtTheseProducts()
     }
 
     /**
-     * Template variable getter. Returns if price alarm is disabled
+     * Template variable getter. Returns if price alarm is enabled
      *
-     * @return int
+     * @return bool
      */
     public function isPriceAlarm()
     {
         // #419 disabling price alarm if article has fixed price
         $article = $this->getProduct();
         if (isset($article->oxarticles__oxblfixedprice->value) && $article->oxarticles__oxblfixedprice->value) {
-            return 0;
```

Comment:
I would return this directly...

         $article = $this->getProduct();
       return isset($article->oxarticles__oxblfixedprice->value) && $article->oxarticles__oxblfixedprice->value;